index.d.ts

```diff
@@ -0,0 +1,52 @@
+// TypeScript typings for "mocha-testdata@1.1.3".
+
+declare module "mocha-testdata" {
```

Comment:
However that Knockout definitions file gets wrapped up with external module spec by `typings`-the-library. You need the external module for TypeScript to generate `require` call.